react-native-libraries.json

```
@@ -13123,6 +13123,39 @@
     "ios": true,
     "android": true,
     "expoGo": true,
+  },
```

```suggestion
    "newArchitecture": true
  },
```

Looks like you have removed new architecture flag from previous entry, and now trailing comma on Expo Go line throws.